integration/service/create_test.go

```diff
@@ -0,0 +1,113 @@
+package service
+
+import (
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/api/types/swarm"
+	"github.com/docker/docker/integration-cli/request"
+	"github.com/gotestyourself/gotestyourself/poll"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+func TestSwarmLBSandbox(t *testing.T) {
```

Comment:
The test name should match the endpoint that is being tested, so something like `TestCreateWithLBSandbox`

Reply:
Fixed